src/core_atmosphere/physics/mpas_atmphys_driver_pbl.F

```diff
@@ -1499,6 +1563,14 @@ subroutine driver_pbl(itimestep,configs,mesh,sfc_input,diag_physics,tend_physics
                   bl_mynn_cloudmix   = bl_mynn_cloudmix    ,                                              &
                   bl_mynn_mixqt      = bl_mynn_mixqt       ,                                              &
                   bl_mynn_ess        = bl_mynn_ess         ,                                              &
+                  mix_chem = .true.,   &
```

Comment:
should this be hardwired to true?

Reply:
no, this can not be hard-wired to true because this will add extra computation (nonlocal transport of zeros) in the MYNN-EDMF even when there is no nonzero chemical species to transport.

Reply:
Thanks Mike and Joe. I will address this depending on the available chemical species.